.circleci/config.yml

```diff
@@ -20,6 +20,22 @@ jobs:
       - store_test_results:
           path: junit-results.xml
   testcompute:
+    docker:
+      - image: cimg/python:3.10
+    resource_class: small
+    environment:
+      OMP_NUM_THREADS: "1"
+      OPENBLAS_NUM_THREADS: "1"
+      MKL_NUM_THREADS: "1"
+      VECLIB_MAXIMUM_THREADS: "1"
+      NUMEXPR_NUM_THREADS: "1"
+    steps:
+      - checkout
+      - run: pip install . -r requirements-dev.txt
+      - run: pytest -rw --durations=5 --junitxml=junit-results.xml test/testcompute/
+      - store_test_results:
+          path: junit-results.xml
+  testviews:
```

Comment:
please change the parallelism of `testviews` to 2

https://app.circleci.com/pipelines/github/sigopt/libsigopt/111/workflows/160cd047-3bff-4659-95bc-e90e3eb243ca/jobs/336/timing

Reply:
yea, was about to

Reply:
and/or break up the slow test file `test/testviews/rest/gp_next_points_test.py`

Reply:
I think it is fine for now (<2 min for each parallel copy). can revisit breaking up this test later if it becomes too cumbersome.